includes/blocks/class-sensei-blocks.php

```diff
@@ -43,10 +43,8 @@ class Sensei_Blocks {
 
 	/**
 	 * Sensei_Blocks constructor.
-	 *
-	 * @param Sensei_Main $sensei Sensei instance.
 	 */
-	public function __construct( Sensei_Main $sensei ) {
```

Comment:
It's not related to this PR. It's something that we probably missed in the past while removing it.